docs/consensus/pot.md

```diff
@@ -0,0 +1,58 @@
+A single timekeeper is sufficient for the security of the protocol, but for robustness and decentralization there will be multiple timekeepers running in parallel. We invite interested parties to run timekeeper component on their nodes to ensure the security and decentralization of the protocol. Domain operators may be more suited for the task since they likely already have powerful hardware.
+
+The timekeepers start the Proof-of-Time chain starts at the genesis time of the Subspace consensus chain. The input to the first slot is a random seed which will be publicly announced at launch to ensure equal opportunity. For each subsequent slot, the output of the previous slot serves as the input. By chaining the outputs, the timekeepers enforce sequentiality and prevent skipping ahead in time.
+
+## Randomness Beacon
+
+The fact that we have a sequence of random values coming from the Proof-of-Time evaluation allows us to use it
+as a source of randomness for block production. Since we target a block challenge every second, we can set the delay function evaluation to output a proof every second. Then for every time slot, timekeepers will evaluate the delay function for a set number of iterations to generate fresh global randomness. They then announce the output to the network, which is used by farmers to determine the next block proposer.
+
+<!-- ![ProofOfTimeChallenges](../../../src/Images/PoTChallenges.png) -->
+
+Farmers receive fresh randomness from timekeepers, verify it and scan their plots to see if they contain any chunks of history close enough to the challenge threshold to claim the block. Farmers with the correct chunks provide a proof-of-space for those, propose a block and earn rewards. The randomness is revealed a few slots in advance to ensure every farmer on the network has had enough time to receive it, scan their plots and submit the proof-of-space if they win. The farmers include PoT outputs in the block header, and the PoT chain is persisted in the consensus chain in this way.
+
+Every 50 blocks, entropy from the consensus chain is injected back into the PoT chain. The injection takes the farming solution and PoT output from a deep consensus block header as the new input for the delay function. Injection also prevents an adversary from simulating a PoT chain without also having to simulate a consensus chain fork. Forks in the consensus chain will result in a different sequence of PoT outputs. Hence, an attacker that forks the chain in some historical point, will have to physically run the PoT algorithm. 
```

Comment:
> Injection also prevents an adversary from simulating a PoT chain without also having to simulate a consensus chain fork

It's the other way around: it prevents simulating a new consensus fork with the old pot chain